### PR TITLE
LSP3 review: fixed typos, improved wording

### PR DESCRIPTION
**Pull request by JeneaVranceanu**

### CODE REVIEW

File: LSPs/LSP-3-UniversalProfile-Metadata.md [hunk 1]
```diff
@@ -186,8 +186,8 @@ For more infos about how to access each index of the `LSP3IssuedAssets[]` array,
 
 ## Rationale
 
-Universal Profiles metadata is important to create verifiable public account that are the source of asset issuance,
-or a verifiable public appearance. This metadata dos not need to belong to a real world person, but gives the account a "face".
+Universal Profile's metadata is important for creating a verifiable public account that is the source of asset issuance,
+or a verifiable public appearance. This metadata does not need to belong to a real world person, but gives the account a "face".
```
Comment (JeneaVranceanu) [inline LSPs/LSP-3-UniversalProfile-Metadata.md:190]: @CJ42 How do you think will it be better to end the last sentence with `... gives the account a "face" that humans are able to recognize.` or shortly `... a "face" to recognize.`?

One of the primary purposes of the metadata and LSP3 profile is to make it look and feel like web2 where people used to upload their photos so they could recognize each other by their faces and names.

Comment (CJ42) [reply LSPs/LSP-3-UniversalProfile-Metadata.md:190]: @JeneaVranceanu maybe `gives the account a "recognisable face"`?

Comment (JeneaVranceanu) [reply LSPs/LSP-3-UniversalProfile-Metadata.md:190]: That's better!

**Comment by JeneaVranceanu** [comment]
Will reopen this PR in lukso-network/LIPs repo.

**Pull request closed without merging**